src/lib/components/FlowDiagramExplorer/flow-diagram-explorer.stories.tsx

```diff
@@ -84,6 +84,15 @@ Simple.args = {
     width: "100%",
     height: "600px",
     diagramConfig: defaultDiagramConfig,
+    animationsOn: true,
+    flowStyles: {
+        "fuel-flow": {
+            strokeColor: "green",
+            strokeStyle: "3 3",
+            strokeWidth: 3,
+            arrowHeadSize: 5,
+        },
+    },
```

Comment:
Does not seem to be active for the Simple example in storybook. Fuel colors not active